src/query_builder.erl

```diff
@@ -13,39 +23,43 @@ lookup_query({in, Collection, Metric}) when is_list(Metric) ->
 lookup_query({in, Collection, Metric, Where}) when is_list(Metric) ->
     lookup_query({in, Collection, dproto:metric_from_list(Metric), Where});
 lookup_query({in, Collection, Metric}) ->
-    Query = ["SELECT DISTINCT bucket, key ",
-             "FROM ", ?METRIC_TABLE, " ",
-             "WHERE collection = $1 and metric = $2"],
-    Values = [Collection, Metric],
-    {ok, Query, Values};
+    Q = ["SELECT DISTINCT bucket, key ",
+         "FROM metrics "
+         "WHERE collection = $1 "
+            "AND metric = $2 "],
+    Vs = [Collection, Metric],
+    {ok, Q, Vs};
 lookup_query({in, Bucket, Metric, Where}) ->
-    Query = ["SELECT DISTINCT bucket, key ",
-             "FROM ", ?METRIC_TABLE, " ",
-             "WHERE collection = $1 AND metric = $2 ",
-             "AND id IN "],
+    Q = ["SELECT DISTINCT bucket, key ",
```

Comment:
this mixes two ways of writing this, one the string concatination the other an iolist, it'd be better to use one of those not both. Given there are no more fields from a -defie, it would probably best just to make it 1 string